collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -235,6 +244,76 @@
 """,
 }
 
+QUERY_BACKEND_USAGE = {
+    DEFAULT: """
+SELECT
+    COUNT(1) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - COUNT(1) AS available
+FROM pg_catalog.pg_stat_activity
+WHERE backend_type IN ('client backend', 'background worker');
+""",
+    V10: """
+SELECT
+    SUM(s.conn) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - SUM(s.conn) AS available
+FROM (
+    SELECT 's' as type, COUNT(1) as conn
+    FROM pg_catalog.pg_stat_activity
+    WHERE backend_type IN ('client backend', 'background worker')
+    UNION ALL
+    SELECT 'r', COUNT(1) 
+    FROM pg_catalog.pg_stat_replication
+) as s;
+""",
+    V92: """
+SELECT
+    SUM(s.conn) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - SUM(s.conn) AS available
+FROM (
+    SELECT 's' as type, COUNT(1) as conn
+    FROM pg_catalog.pg_stat_activity
+    WHERE query NOT LIKE 'autovacuum: %%'
+    UNION ALL
+    SELECT 'r', COUNT(1) 
+    FROM pg_catalog.pg_stat_replication
+) as s;
+""",
+    V91: """
+SELECT
+    SUM(s.conn) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - SUM(s.conn) AS available
+FROM (
+    SELECT 's' as type, COUNT(1) as conn
+    FROM pg_catalog.pg_stat_activity
+    WHERE current_query NOT LIKE 'autovacuum: %%'
+    UNION ALL
+    SELECT 'r', COUNT(1) 
+    FROM pg_catalog.pg_stat_replication
+) as s;
+""",
+    V82: """
+SELECT
+    COUNT(1) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - COUNT(1) AS available
+FROM pg_catalog.pg_stat_activity
+WHERE current_query NOT LIKE 'autovacuum: %%';
+""",
+    V72: """
+SELECT
+    COUNT(1) as used,
+    current_setting('max_connections')::int - current_setting('superuser_reserved_connections')::int
+    - COUNT(1) AS available
+FROM pg_catalog.pg_stat_activity s
+JOIN pg_catalog.pg_database d ON d.oid = s.datid
+WHERE d.datallowconn;
```

Comment:
I think we do not need this. I suppose it is from 
https://github.com/OPMDG/check_pgactivity/blob/a09b8b823bd0529be93d846b34b2c582f7769477/check_pgactivity#L2081

Reply:
Yes, it's takent from there. I kept it because they had it only for versions before v8.2 and then removed it, so I was assuming there was a valid reason for it. I have no way to test such old PG versions.

Reply:
Regarding versions, there's probably no real value in capping PG version support so aggressively. There's still plenty of pre-9.5 PGs running in production today, and the monitoring code won't need to change that much.